### ISSUE TITLE
[!!] Sum logic in the "Show Report" box

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-atag-report-tool#168]
(This is specifically for SC where you can select the conformance level you are measuring. It is tied to the overall level conformance target selected in the About page.)

The logic used in calculating progress in the Show Report box is not correct when calculating SCs that are set to measure against "AAA" and the overall target is set at "AA". And other scenarios similar. To me this cause a bit of confusion until I tested and figured out what was going on.

Some SCs the user can select the Level they are choosing to measure against. This is especially prevalent in the B1 section. The default level for me was set to level AA. When I selected Passed, Failed, etc. it was counted in that section in the Show Report box. But if I switched it to measure against level AAA, then it was not counted at all. So even though I selected Passed, it was not counted.

Also seems to not be counted in the summary box on the report page. However, it is removed from the not checked section of the report page, so that worked.

**Comment by hidde** [comment]
This is on purpose: the totals only count the results that are _within_ conformance target. You can still set those outside conformance target, but they don't count. Most  SCs outside target will be collapsed, for those the behavior will probably make most sense. The ones where you can select multiple levels are not collapsed, because there applicability depends on the level that is set, so I understand it is a little less straightforward there. As soon as you decide to evaluate it _as_ a target that is higher than your overall conformance target, it stops being counted.

The alternative, that we do count SCs that are outside conformance target, might be more confusing, because then it could look like you've done 100% or more, when you've only done a lot of outside-target SCs, but not the ones that were inside target.

In other words, I feel the current method of counting is the least bad option. I think there was some discussion around whether to explain all of this in the Overview page, but I worry that the explanation is fairly complicated (as above, haha), so it may needlessly confuse people. (My assumption is that few people will change the conformance target or review things outside target)

**Comment by bakkenb** [comment]
Although I am still worried a few people may be thrown for a loop by this, it only impacts the count in the Your Report box, and only when they start changing conformance level of individual SCs. 

I agree it would be to difficult to explain without causing even more confusion. So I am okay with this reasoning and okay to have this issue closed.

**Comment by hidde** [comment]
Thank you, Brent! It's good to have this logged for feature reference.
